types/stripe/index.d.ts

```diff
@@ -4785,8 +4795,8 @@ declare namespace Stripe {
              * you created a charge with the capture option set to false. Uncaptured payments expire exactly seven days after they are
              * created. If they are not captured by that point in time, they will be marked as refunded and will no longer be capturable.
              */
-            capture(id: string, options: HeaderOptions, response?: IResponseFn<charges.ICharge>): Promise<charges.ICharge>;
-            capture(id: string, response?: IResponseFn<charges.ICharge>): Promise<charges.ICharge>;
+            capture(id: string, data: charges.IChargeCaptureOptions, options: HeaderOptions, response?: IResponseFn<charges.ICharge>): Promise<charges.ICharge>;
```

Comment:
Just confirming - so the previous two overloads weren't valid, and the `data` parameter requires an object that unconditionally has an `amount` property?

Reply:
I don't know how I missed this but no, the `amount` property is optional, and the entire data object is. I spent a lot of time digging in the code for the stripe library and both `data` and `options` are optional, they distinguish them by looking at which keys are present

@lauvigne could you add `?` to both `data` and `amount`?